content/general/concepts/software-development-life-cycle/software-development-life-cycle.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'Software Development Life Cycle'
+Description: 'The Software Development Life Cycle (SDLC) is a process that is used to efficiently produce cost-effective and high-quality software through a series of structured steps or phases.'
+Subjects:
+  - 'Mobile Development'
+  - 'Web Development'
+Tags:
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Software Development Life Cycle (SDLC)** is a process that is used to efficiently produce cost-effective and high-quality software through a series of structured steps or phases. Essentially, it is a standard industry approach to software development that outlines an entire project and timeline.
```

Comment:
I think this should be reframed to be more agnostic relative to the potential cost and quality benefits.  SDLC is just a formal or "structured process" for the development and deployment of software. The use of any model doesn't guarantee savings or efficiency, I think that the relative benefits are probably highly dependent on how a model is implemented. I would suggest that the concrete deliverables of implementing a formal process is direction to the team in the development of a product, as well as documentation.